static/css/components/badges.sass

```diff
@@ -29,4 +29,10 @@
   &.bot
     background: url('/static/images/badge_bot.png')
     background-size: 100%
+  &.vip
+    background: url('/static/images/badge_vip.png')
+    background-size: 100%
```

Comment:
```suggestion
    background-size: 100%
    background-color: rgb(224, 5, 185)
```

Reply:
yeah I just realized that this is not actually the case. The image has the background "baked in".

I was using inspect element on the Twitch website. Apparently the FFZ browser extension changes the standard badges to SVG ones with a CSS background.

Reply:
So should I also do this to be consistent? Or just go with what I have?

Reply:
keep it as it was before